README.md

```diff
@@ -0,0 +1,55 @@
+# CBL/reduce
+A family of scripts to prepare and execute reduction (cvise, llvm-reduce) workflows.
+
+Currently, there are two scripts:
+
+1) prep
+> Prepare a target file for reduction (see Basic Usage)
+
+2) flags
+> Minimize the compiler flags required to reproduce behavior. Use after prep
+
+## Dependencies
+* Python 3.11
+* [icecream](https://pypi.org/project/icecream/)
+* [C-Vise](https://github.com/marxin/cvise)
+
+## Basic Usage
+
+`$ python reduce.py -h`
+
+### prep
+> Get help with prep
+
+`$ python reduce.py prep -h`
```

Comment:
needs more Preparation-H jokes

Reply:
gpt was glad to help.

```
Sure, here are some humorous lines you might find amusing:                                                                                                                                                                                      ▎   ▎ 
 ▊ ▊                                                                                                                                                                                                                                                    ▎   ▎ 
 ▊ ▊      1. "No side effects guaranteed, only code effects!"                                                                                                                                                                                           ▎   ▎ 
 ▊ ▊      2. "Coding with 'prep -h' may relieve symptoms of buggy code, unlike Preparation-H!"                                                                                                                                                          ▎   ▎ 
 ▊ ▊      3. "Warning: Excessive use of 'prep -h' may result in becoming a Python expert, not a hemorrhoid treatment expert!"                                                                                                                           ▎   ▎ 
 ▊ ▊      4. "Relieves code constipation. More effective than Preparation-H!"                                                                                                                                                                           ▎   ▎ 
 ▊ ▊      5. "For highly irritating bugs, we recommend 'prep -h', but for everything else, there's Preparation-H."                                                                                                                                      ▎   ▎ 
 ▊ ▊      6. "Helps your code get back on track. Soothing as Preparation-H but strictly for programmers!"                                                                                                                                               ▎   ▎ 
 ▊ ▊      7. "Unlike Preparation-H, 'prep -h' doesn't require application once every 4 hours!"                                                                                                                                                          ▎   ▎ 
 ▊ ▊      8. "'prep -h' - the only relief you need from memory leaks, not other leaks!"                                                                                                                                                                 ▎   ▎ 
 ▊ ▊      9. "'prep -h', the only ointment your code needs!"                                                                                                                                                                                            ▎   ▎ 
 ▊ ▊     10. "Side effects may include cleaner code and an uncontrollable urge to program! Far better than Preparation-H."
```

Reply:
Personal favorite: 

"Unlike Preparation-H, 'prep -h' doesn't require application once every 4 hours!"